Registry/AvailableModule.cs

```diff
@@ -43,10 +44,10 @@ public void Remove(Version version)
         ///     If no KSP version is supplied, we return the latest release of that module for any KSP.
         ///     Returns null if there are no compatible versions.
         /// </summary>
-        public CkanModule Latest(KSPVersion ksp_version = null)
+        public CkanModule Latest(KSPVersion ksp_version = null, RelationshipDescriptor relationship=null)
```

Comment:
Any chance we can get what `relationship` is used for in the method documentation?